demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -0,0 +1,183 @@
+
+    def ident_changed_file_type(self, file_path):
+        _file_type = None
+        file_name = 'N/A'
+        if self.pack in file_path:
+            _file_path = self.find_corresponding_yml(file_path)
+            file_name = self.get_display_name(_file_path)
+            if 'Playbooks' in file_path and ('TestPlaybooks' not in file_path):
+                _file_type = 'Playbook'
+            elif 'Integration' in file_path:
+                _file_type = 'Integration'
+            elif 'Script' in file_path:
+                _file_type = 'Script'
+            # incident fields and indicator fields are using the same scheme.
+            elif 'IncidentFields' in file_path:
+                _file_type = 'IncidentFields'
+            elif 'IncidentTypes' in file_path:
+                _file_type = 'IncidentTypes'
+            elif 'Classifiers' in file_path:
+                _file_type = 'Classifiers'
+            elif 'Layouts' in file_path:
+                _file_type = 'Layout'
+
+        return file_name, _file_type
+
+    def bump_version_number(self, pre_release: bool = False):
+
+        new_version = None  # This will never happen since we pre-validate the arguement
+        data_dictionary = get_json(self.metadata_path)
+        if self.update_type == 'major':
+            version = data_dictionary.get('currentVersion', '99.99.99')
+            version = version.split('.')
+            version[0] = str(int(version[0]) + 1)
+            version[1] = '0'
+            version[2] = '0'
+            new_version = '.'.join(version)
+        elif self.update_type == 'minor':
+            version = data_dictionary.get('currentVersion', '99.99.99')
+            version = version.split('.')
+            version[1] = str(int(version[1]) + 1)
+            version[2] = '0'
+            new_version = '.'.join(version)
+        # We validate the input via click
+        elif self.update_type == 'revision':
+            version = data_dictionary.get('currentVersion', '99.99.99')
+            version = version.split('.')
+            version[2] = str(int(version[2]) + 1)
+            new_version = '.'.join(version)
+        if pre_release:
+            new_version = new_version + '_prerelease'
+        data_dictionary['currentVersion'] = new_version
+
+        if self._does_pack_metadata_exist():
+            with open(self.metadata_path, 'w') as fp:
+                json.dump(data_dictionary, fp, indent=4)
+                print_color(f"Updated pack metadata version at path : {self.metadata_path}",
+                            LOG_COLORS.GREEN)
+        return new_version
+
+    @staticmethod
+    def check_rn_dir(rn_path):
+        if not os.path.exists(os.path.dirname(rn_path)):
+            try:
+                os.makedirs(os.path.dirname(rn_path))
+            except OSError as exc:  # Guard against race condition
+                if exc.errno != errno.EEXIST:
+                    raise
+
+    def build_rn_template(self, changed_items: dict):
```

Comment:
maybe I am missing her something, but I dont understand how existing release notes file is handled. it seems that if we have already an existing one, an there is an existing line under "integration" or any other part, it will overwrite it.

Reply:
We should never have an existing release note for that version, right?